### PR TITLE
252 spacing audit demo page

### PR DESCRIPTION
**Pull request by abbeyhrt**
To aid in the Spacing Audit #252 

This adds a Spacing Audit page that will ideally show all the possible spacing variations for different components in the Theme. Let me know if something needs to be added to it! 

Link to new page in deployment: 
https://gatsby-theme-carbon-bpyj0kb57.now.sh/components/SpacingAudit

### CODE REVIEW

**Comment by alisonjoseph** [comment]
This looks great, I think we can leave it in the PR for our meeting and then close w/ out merging.

File: packages/example/src/pages/components/SpacingAudit.mdx [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+</Column>
+</Row>
+
+</Tab>
+
+</Tabs>
+
+I'm a paragraph below Tabs. Lorem ipsum dolor sit amet, consectetur adipiscers elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+
+## Paragraph and Code Snippet
```
Comment (abbeyhrt) [inline packages/example/src/pages/components/SpacingAudit.mdx]:
```suggestion
## Paragraph and Code
```

**Comment by vpicone** [comment]
Woah! Awesome! We could also just not add it to the side nav and let it be a 'hidden' page we use to test things. What do you think @alisonjoseph ?

**Comment by alisonjoseph** [comment]
@vpicone  good idea 👍

**Comment by vpicone** [comment]
@abbeyhrt do you mind taking the page off the sidenav? It'll just be a hidden page for us to test stuff.

**Comment by abbeyhrt** [comment]
@alisonjoseph I removed the page from the `nav-items.yaml` and created a test directory for the spacing audit to live in if we want to merge this in that way? I don't mind if it lives in PR land either, up to you all!

**Pull request merged**